intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -110,14 +113,18 @@ def train(data):
 #   After profiling, result files will be saved into the ``./log/resnet18`` directory.
 #   Specify this directory as a ``logdir`` parameter to analyze profile in TensorBoard.
 # - ``record_shapes`` - whether to record shapes of the operator inputs.
+# - ``profile_memory`` - Track tensor memory allocation/deallocation.
+# - ``with_stack`` - Record source information (file and line number) for the ops.
+#   If the TensorBoard is launched in VSCode, clicking a stack frame will navigate to the specific code line.
```

Comment:
It'd be great to link to this doc: https://code.visualstudio.com/docs/python/pytorch-support